modules/commons/src/main/java/org/apache/synapse/commons/json/JsonDataSourceImpl.java

```diff
@@ -109,7 +112,7 @@ public JsonDataSourceImpl(InputStream inputStream) {
             }
         } catch (IOException e) {
             logger.error("Could not create a JSON data source from the input stream. "
-                         + e.getLocalizedMessage());
+                    + e.getLocalizedMessage());
```

Comment:
What would be the reason for using e. getLocalizedMessage() instead of e.getMessage()? Do we support translations?